### ISSUE TITLE
[Extractor] 'No such host is known' error when running

### ISSUE THREAD

**Issue by erwinkramer** [Azure/azure-api-management-devops-resource-kit#259]
When running the extractor, either latest from source or the last beta from releases, i get: 

```
Connecting to xyz API Management Service on abc Resource Group ...
Executing full extraction ...
------------------------------------------
Extracting global service policy from service
Error occured: No such host is known
No such host is known
```

Anyone know what's going on? I'm running behind a corporate proxy but logging into Azure and restoring dotnet packages is no problem.

**Comment by miaojiang** [comment]
It seems the tool is not able to connect to the Azure Resource Manager REST API. Please try calling the API directly on the box, you should get the same error.

**Comment by UmairSyed** [comment]
@miaojiang I ran it just now and did not find any issues.

**Comment by miaojiang** [comment]
Thanks @UmairSyed. I believe the error is not related to the tool. Closing now.

**Comment by erwinkramer** [comment]
@miaojiang  and @UmairSyed I did manage to make it work, but it's kind of a work around.

When i boot up Fiddler and enable Capture Traffic, the Extractor just works. 

When i disable Capture Traffic and try out the ARM REST API [in the browser](https://docs.microsoft.com/en-us/rest/api/apimanagement/2019-01-01/apis/listbyservice#code-try-0) or in the same PowerShell session as `Invoke-WebRequest`, it works just fine. 

My assumption is something with certificates going wrong, and it's hiding that error. I had to make some additions in the `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\Lib\site-packages\certifi\cacert.pem` file before, to make `az login` for the Azure CLI work. Also the Azure CLI with Capture Traffic disabled is just working fine too. 

On this [Telerik blog post](https://webcache.googleusercontent.com/search?q=cache:sd1jYbvlpRAJ:https://www.telerik.com/blogs/help!-running-fiddler-fixes-my-app-+&cd=1&hl=nl&ct=clnk&gl=nl) are some nice tips on the same issue, but i'm not seeing anything i can do on my side to solve the issue.
